views/js/controller/creator/views/section.js

```diff
@@ -245,7 +245,7 @@ define([
 
                             //the itemRef should also "inherit" the categories set at the item level
                             const categories = sectionCategory.getCategories(sectionModel);
```

Comment:
`categories` isn't used, no need this line
<img width="554" alt="image" src="https://user-images.githubusercontent.com/25976342/163175369-872ddcb4-5625-4dfa-8de9-b7134b8c6881.png">